README.md

```diff
@@ -83,6 +83,8 @@ kubectl exec -it ${ST2CLIENT} /bin/bash
 ### [st2web](https://docs.stackstorm.com/latest/reference/ha.html#nginx-and-load-balancing)
 st2web is a StackStorm Web UI admin dashboard. By default, st2web K8s config includes a Pod Deployment and a Service.
 `2` replicas (configurable) of st2web serve the web app and proxy requests to st2auth, st2api, st2stream.
+By default, st2web uses HTTP instead of HTTPS. Pass in ST2WEB_HTTPS environment variable with a non-zero value
+(e.g. "1") and ensure SSL certificates are configured.
```

Comment:
I don't think we need to mention `ST2WEB_HTTPS` or certificates here as it was designed for possible docker-compose use cases outside of K8s where users don't have concept of Ingress layer.